src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
 export interface Figure {
-
+  shape: string,
+  color: string,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Invalid Number');
+    }
+
+    if (a + b <= c || b + c <= a || a + c <= b) {
+      throw new Error('sides 1, 2 and 3 can`t form a triangle');
+    }
+  }
+
+  getArea(): number {
+    const per: number = (this.a + this.b + this.c) / 2;
+    const square = Math
+      .sqrt(per * (per - this.a) * (per - this.b) * (per - this.c));
 
+    return Math.floor(square * 100) / 100;
+  }
 }
 
-export class Circle {
+export class Circle implements Figure {
+  shape: string = 'circle';
 
+  constructor(
+    public color: string,
+    public radius: number,
+  ) {
+    if (radius <= 0) {
+      throw new Error('Number is not correct');
```

Comment:
Would be great to throw more descriptive errors
![image](https://github.com/mate-academy/ts_figures_classes/assets/96409563/6f7a74cb-b5c7-47e4-93a4-946e1252b2d3)